### ISSUE TITLE
ServiceNotFoundException: lexik_jwt_authentication.handler.authentication_success

### ISSUE THREAD

**Issue by Lemonify** [lexik/LexikJWTAuthenticationBundle#423]
Hello,

I followed a documentation over here: https://github.com/lexik/LexikJWTAuthenticationBundle/blob/master/Resources/doc/7-manual-token-creation.md and tried to implement the last part with `handleAuthenticationSuccess` method. However, it seems it's not working in Symfony 4 anymore.

`The "lexik_jwt_authentication.handler.authentication_success" service or alias has been removed or inlined when the container was compiled. You should either make it public, or stop using the container directly and use dependency injection instead.`

So I tried to inject it in controller's constructor, but...

`Cannot autowire service "App\Controller\UserController": argument "$authenticationSuccessHandler" of method "__construct()" references class "Lexik\Bundle\JWTAuthenticationBundle\Security\Http\Authentication\AuthenticationSuccessHandler" but no such service exists. It cannot be auto-registered because it is from a different root namespace.`

It was necessary to add these four lines to config/services.yaml:
```
Lexik\Bundle\JWTAuthenticationBundle\Services\JWTManager:
        autowire: true

Lexik\Bundle\JWTAuthenticationBundle\Security\Http\Authentication\AuthenticationSuccessHandler:
        autowire: true
```
Is this actually the right way of doing this? It feels a bit hacky to me, but I am not really a Symfony expert :) In any way, the documentation seems to be obsolete in this case.

**Comment by chalasr** [comment]
Thanks for reporting this bug, I will try to find the best approach for fixing it. Keep you informed

**Comment by chalasr** [comment]
@brunoreis This bundle works fine with symfony 4, it's tested and I use it everyday. The doc needs some adjustments regarding naming of config files in code examples, but there is no blocker, a flex recipe is available under the `jwt-auth` alias. 
About this issue, the deprecation seems legit, dependency injection should be used. The doc needs to be fixed to not call `get()` on a container-aware controller, not the code.

**Comment by brunoreis** [comment]
Thanks. In fact I installed it using the recipe and I'm using it successfully already. Thanks a lot for this.

**Comment by chalasr** [comment]
Fixed by #465

**Comment by CuBeR116** [comment]
> Fixed by #465

In Symfony 6.0 this problem returned

**Comment by chalasr** [comment]
Please open a new issue with a full reproducer.
